src/rgw/rgw_rados.cc

```diff
@@ -6877,7 +6889,11 @@ int RGWRados::bucket_index_read_olh_log(const RGWBucketInfo& bucket_info, RGWObj
   int ret =
     bs.init(obj_instance.bucket, obj_instance, nullptr /* no RGWBucketInfo */);
   if (ret < 0) {
-    ldout(cct, 5) << "bs.init() returned ret=" << ret << dendl;
+    if (ret == -ENOENT) {
```

Comment:
here again I'm unsure whether to trust that the current request has had no side effect/is safely retryable